### PR TITLE
FT0: time offset calibration workflow for global run

### PR DESCRIPTION
**Pull request by afurs**
Second stage for FT0 time offset calibration workflow within global run.
First stage: https://github.com/AliceO2Group/O2DPG/pull/707

### CODE REVIEW

File: prodtests/full-system-test/calib-workflow.sh [hunk 1]
```diff
@@ -55,6 +56,9 @@ if workflow_has_parameter CALIB_PROXIES; then
   if [[ ! -z $CALIBDATASPEC_ZDC_TF ]]; then
     add_W o2-dpl-output-proxy "--dataspec \"$CALIBDATASPEC_ZDC_TF\" $(get_proxy_connection zdc_tf output)" "" 0
   fi
+  if [[ ! -z $CALIBDATASPEC_FT0_TF ]]; then
```
Comment (davidrohr) [inline prodtests/full-system-test/calib-workflow.sh]: please merge ZDC and FIT proxies into a single fwd_tf proxy

File: prodtests/full-system-test/aggregator-workflow.sh [hunk 2]
```diff
@@ -270,6 +272,9 @@ if [[ $AGGREGATOR_TASKS == FORWARD_TF || $AGGREGATOR_TASKS == ALL ]]; then
   if [[ $CALIB_ZDC_TDC == 1 ]]; then
     add_W o2-zdc-tdccalib-workflow
   fi
+  if [[ $CALIB_FT0_TIMEOFFSET == 1 ]]; then
+    add_W o2-calibration-ft0-time-offset-calib "--tf-per-slot $FT0_TIMEOFFSET_TF_PER_SLOT"
```
Comment (chiarazampolli) [inline prodtests/full-system-test/aggregator-workflow.sh]: Hi @afurs . 
As discussed via mail, here you could add your extra config option:
```
add_W o2-calibration-ft0-time-offset-calib "--tf-per-slot $FT0_TIMEOFFSET_TF_PER_SLOT" "FT0CalibParam.mNExtraSlots=0;FT0CalibParam.mRebinFactorPerChID[180]=4;"

Comment (afurs) [reply prodtests/full-system-test/aggregator-workflow.sh]: Hello @chiarazampolli , I added default configKeyValue as arg for `add_W()`

**Comment by afurs** [comment]
Hello @chiarazampolli can you merge this PR?

**Comment by chiarazampolli** [comment]
I can't, we need @davidrohr , or @shahor02 .

**Pull request merged**